.pre-commit-config.yaml

```diff
@@ -24,9 +24,10 @@ repos:
     hooks:
       - id: isort
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.790
+    rev: v0.910
     hooks:
       - id: mypy
+        additional_dependencies: [types-all]
```

Comment:
This was the cleanest solution I could come up with to automatically populate stubs for mypy pre-commit hooks: https://github.com/pre-commit-ci/issues/issues/69#issuecomment-880033342

Reply:
@villebro this is what we did at Airbnb as well. It's a real shame that Mypy doesn't install these by default if needed.